File: Fundamentos de Desenvolvimento com C#/TP3/FormasGeometricas/Esfera.cs
namespace FormasGeometricas {
    public class Esfera {
        // Atributos
        public double Raio { get; set; }

        // Construtor
        public Esfera(double raio) {
            Raio = raio;
        }

        // Método para calcular área do círculo
        public double CalcularVolume() {
            return (4.0 / 3.0) * Math.PI * Math.Pow(Raio, 3);
        }
    }
}

File: Fundamentos de Desenvolvimento com Java/TP3/src/FormasGeometricas/Esfera.java
package FormasGeometricas;

public class Esfera {
    private double raio;

    public Esfera(double raio) {
        this.raio = raio;
    }

    public double getRaio() {
        return raio;
    }

    public void setRaio(double raio) {
        this.raio = raio;
    }

    public double calcularVolume() {
        return (4.0 / 3.0) * Math.PI * Math.pow(raio, 3);
    }
}
